uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -178,6 +181,20 @@ define([
     }
 
     /**
+     * Opens the leave request popup
+     *
+     * @param {Object} event
+     * @param {Object} leaveRequest
+     * @param {String} leaveType
+     * @param {String} selectedContactId
+     * @param {Boolean} isSelfRecord
+     */
+    function openLeavePopup (event, leaveRequest, leaveType, selectedContactId, isSelfRecord) {
+      event.stopPropagation();
```

Comment:
ok at least explain what it does in the function documentation, it's not self explanatory at all